operators/full-mentions.js

```diff
@@ -2,6 +2,10 @@ const { deferred } = require('jitdb/operators')
 
 module.exports = function mentions(key) {
   return deferred((meta, cb) => {
-    meta.db2.getIndexes().fullMentions.getMessagesByMention(key, meta.live, cb)
+    meta.db2.onDrain('fullMentions', () => {
+      meta.db2
+        .getIndexes()
+        .fullMentions.getMessagesByMention(key, meta.live, cb)
+    })
```

Comment:
This one was actually a great idea! The custom operator can do the onDrain for the specific index that it is meant to, in this case `full-mentions`.